config/locales/en.yml

```diff
@@ -278,17 +278,21 @@ en:
       license_context: 'This means that you can use, share and adapt it, as long as you give proper attribution, provide a link to the license, and indicate if changes were made to the image. See the %{link} for more details.'
       license_context_link: 'full license'
       portraits_heading: 'Cropped portraits'
-      download_jpeg: 'download jpeg'
+      download_jpeg: 'download'
```

Comment:
a similar value exists of line 307, you could try having a single key:value pair for the string 'download'